cdap-docs/developers-manual/source/data-exploration/custom-datasets.rst

```diff
@@ -2,10 +2,10 @@
     :author: Cask Data, Inc.
     :copyright: Copyright © 2014-2015 Cask Data, Inc.
 
-.. _data-exploration:
+.. _custom-dataset-exploration:
```

Comment:
If you change the ref, you need to find everywhere that its used and change those. Best just to add another ref and have both of them, unless you want to find all uses of "<data-exploration>".